src/test/java/com/marklogic/kafka/connect/sink/HandleWriteFailureTest.java

```diff
@@ -4,16 +4,21 @@
 import com.marklogic.client.io.DocumentMetadataHandle;
 import org.apache.kafka.common.record.TimestampType;
 import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNull;
 
 public class HandleWriteFailureTest extends AbstractIntegrationSinkTest {
 
+    Throwable reportedException = null;
+    SinkRecord reportedSinkRecord = null;
```

Comment:
We need one more SinkRecord now in this test - a valid one that has valid JSON in it. And we then want to make sure that 2 SinkRecords are sent to the ERR, as we don't know which one caused the failure.